metaci/release/models.py

```diff
@@ -79,6 +104,9 @@ def get_production_date():
     change_case_link = models.URLField(
         _("change case link"), max_length=1024, null=True, blank=True
     )
+    submitted_case = models.BooleanField(
+        _("submitted_case"), default=False, editable=False
+    )
```

Comment:
@jofsky I thought we decided we don't need a submitted_case field, since we can tell whether a case was submitted based on whether it was recorded in the change_case_link field